modules/core/src/main/scala/higherkindness/droste/package.scala

```diff
@@ -9,13 +9,30 @@ import syntax.compose._
 
 object `package` {
 
+  /** Type Alias for `F[A] => A` */
   type Algebra[F[_], A]   = GAlgebra[F, A, A]
+
+  /** Type Alias `Coalgebra[F, A]` is a type alias for `GCoalgebra[F, A, A]`.
```

Comment:
How about just:
"A function `A => F[A]`"

Reply:
Problem is, it is not a function straight away, but a wrapper for a function.